scripts/verify_conda_libmesh.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+# * This file is part of the MOOSE framework
+# * https://www.mooseframework.org
+# *
+# * All rights reserved, see COPYRIGHT for full restrictions
+# * https://github.com/idaholab/moose/blob/master/COPYRIGHT
+# *
+# * Licensed under LGPL 2.1, please see LICENSE for details
+# * https://www.gnu.org/licenses/lgpl-2.1.html
+
+import os
+import sys
+try:
+    from conda_build.metadata import MetaData
+except ImportError:
+    print('conda_build module not available.')
+    sys.exit(1)
+
+
+def main():
+    """Check to see if the user's current conda environment is up to date."""
+
+    # Locate MOOSE directory
+    os.chdir(os.path.abspath(os.path.dirname(__file__)))
+    moose_dir = os.getenv('MOOSE_DIR', os.path.join(os.getcwd(), '..'))
```

Comment:
MOOSE_DIR should be passed into the script. The makefile has the authoritative value for this